src/hooks/useEnableToken.ts

```diff
@@ -20,12 +20,12 @@ export const useEnableTokens = (params: Params): Result => {
   const { enabled: enabledInitial, address: tokenAddress } = params.tokenBalances
   const [enabled, setEnabled] = useState(enabledInitial)
   const [enabling, setEnabling] = useState(false)
-  const [highlight, setHightlight] = useState(false)
-  let unmounted = false
+  const [highlight, setHighlight] = useState(false)
+  const mounted = useRef(true)
```

Comment:
https://reactjs.org/docs/hooks-faq.html#is-there-something-like-instance-variables

Also, the react hooks eslist rule given the following warning when not using a ref:

> Assignments to the 'mounted' variable from inside React Hook useEffect will be lost after each render. To preserve the value over time, store it in a useRef Hook and keep the mutable value in the '.current' property. Otherwise, you can move this variable directly inside useEffect.eslint(react-hooks/exhaustive-deps)